genesis/process/shardGenesisBlockCreator.go

```diff
@@ -298,6 +298,7 @@ func createProcessorsForShardGenesisBlock(arg ArgsGenesisBlockCreator, enableEpo
 		DataPool:           arg.Data.Datapool(),
 		CompiledSCPool:     arg.Data.Datapool().SmartContracts(),
 		NilCompiledSCStore: true,
+		EpochNotifier:      epochNotifier,
```

Comment:
same thing as in metaGenesisBlockCreator.go

Reply:
done, although not needed :D